packages/graphql/src/schema.ts

```diff
@@ -16,12 +16,12 @@ const dirname = typeof __dirname !== 'undefined' ? __dirname : ''
 // for vite
 process.cwd ??= () => ''
 
-const isCodegen = Boolean(process.env.GRAPHQL_CODEGEN)
+const isCodegen = Boolean(process.env.CYPRESS_NEXUS_CODEGEN)
```

Comment:
`CYPRESS_INTERNAL` for Cy build + dev utils. `CYPRESS_` for user-modifiable environment variables.
```suggestion
const isCodegen = Boolean(process.env.CYPRESS_INTERNAL_NEXUS_CODEGEN)
```